era_5g_interface/task_handler.py

```diff
@@ -44,3 +44,20 @@ def store_image(self, metadata: dict, image: np.ndarray) -> None:
         """
 
         pass
+
+    @abstractmethod
+    def store_control_data(self, data: dict) -> None:
```

Comment:
Shouldn't the "data" be of type ControlCommand?

Reply:
Yes, I will change it to ControlCommand.